ec-europa/ecl-toolkit
# feat(flavors): prepare theme for flavors - INNO-927

This pull requests extracts necessary changes from https://github.com/ec-europa/europa-component-library/pull/567

In the theme:
- updated dependencies for ECL
- header and footer use new links
- some regions become dynamic following `project.title` from fractal configuration
- styles are linted through our standards, nothing special or actual changed in the CSS

To test the theme:
- `yarn` to take new dependencies
- `cd examples/basic` to work with the example style guide based on the theme
- `yarn start`

There should be no side effects or buggy things, that's it.

In the ecl-builder, the options parameter has been updated to actually work. Appeared that having the config file in a sub-folder broke the cli capabilities because the `commander` holds the option value in root.

To test this, move the `ecl-builder.config.js` somewhere and use the following command: `npx ecl-builder -c test/ecl-builder.config.js` to validate that the config is actually taken. Without the change in this pull request, you will get an error about config file location regardless of whether and how you use `-c` or `--config`.
